libs/cua-driver/test-harness/run-suite.sh

```
@@ -0,0 +1,89 @@
+
+if [ "$BUILD" -eq 1 ]; then
+  echo "==> building host harness ($HOST)…"
+  bash "$HARNESS_DIR/build/$HOST.sh"
+fi
+
+cd "$RUST_DIR"
+declare -a SUMMARY
+overall=0
+for t in "${TESTS[@]}"; do
+  out=$(cargo test $PROFILE -p cua-driver --test "$t" \
+        ${CARGO_TEST_FILTER:-} -- --ignored --nocapture --test-threads=1 2>&1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)